src/vocabulary/mod.rs

```diff
@@ -13,7 +15,7 @@ use processor::TokenProcessor;
 mod locator;
 mod processor;
 
-/// Vocabulary of an LLM.
+/// Creates `Vocabulary` manually or from defined large language model.
```

Comment:
This looks like a description of a method, not the description of a type.

Reply:
I will update 👍